lib/relationships.js

```diff
@@ -130,3 +155,17 @@ function updateRelation (model, id, data) {
     }
   })
 }
+
+function keyByModel (assocModel, model) {
+  var key = null
+  _.each(assocModel.relations, function (relation) {
+    if (relation.modelTo.modelName === model.modelName) {
+      key = relation.keyFrom
+    }
+  })
+
+  if (key === null) {
+    console.log('Can not find relation for ' + model.modelName)
```

Comment:
should I
`var debug = require('debug')('loopback-component-jsonapi')`
or take it from the options and pass through the chain of functions?

```Javascript
module.exports = function (app, options) {
  var debug = options.debug
  // ...
    relationships(id, data, model, debug)
```